praise-participants.md

```diff
@@ -0,0 +1,60 @@
+## Title
+Praise Participants
+
+## Patlet
+After an inner source contribution it's important to thank the contributor for their time and effort.
+This pattern gives guidance for a way to do so that not only effectively acknowledges the contribution but also endgenders further engagement from the contributor and others.
+
+## Problem
+How can we properly express our gratitude to a contributor for their inner source contribution to a project?
+It can be easy to forget to do so or not know the words or medium to use for adequate effect and sincerity.
+A pattern in this area makes it easy to do and ensures that the message comes across clearly and sincerely.
+
```

Comment:
How about phrasing the problem such that the consequence of not dealing out praise for good work becomes more clear? I'm thinking

- People leave projects because they feel their work is not valued
- Managers will pull contributors from products because they don't know how valuable they are

Reply:
I like your suggestion, Georg. How about:
How can we properly express our gratitude to a contributor for their InnerSource contribution to a project? It can be easy to forget to do so or not know the words or medium to use for adequate effect and sincerity. Consequences of this omission can be serious:

- People leave projects because they feel their work is not valued.
- Managers will pull contributors from products because they don't know how valuable they are.

Reply:
How about expressing these as motivations in favour of praising instead of risks when forgetting praise:

We want contributors to inner source projects remain engaged and active. For that to happen 
* the individual contributor needs to remain motivated, 
* their manager needs transparency over their contributions.
Giving credit publicly is an easy and cheap way to accomplish that.

Reply:
`Praise and thanks are easy, affordable ways to keep contributors and their managers motivated and excited to continue.`